BUILD.bazel

```diff
@@ -18,26 +18,31 @@ gazelle(
     gazelle = "gazelle_bin",
 )
 
+sh_binary(
+    name = "gazelle.check",
```

Comment:
Workflows expects a `//:gazelle.check` target to run that exits non-zero if there are changes